extensions/wikia/VideoHandlers/VideoHandlerHelper.class.php

```diff
@@ -493,4 +493,23 @@ public function updateThumbnailData( $file ) {
 		return $status;
 	}
 
+	/**
+	 * Check if video's provider is supported base on its URL.
+	 * Logic is partially ported from ApiWrapperFactory::getApiWrapper method.
+	 * @param $url
+	 * @return bool
+	 */
+	public function isVideoProviderSupported( $url ) {
+		global $wgVideoMigrationProviderMap;
+
+		$parsed = parse_url( strtolower( $url ) );
```

Comment:
As you are using only one element of the array later on, you might go for `parse_url( strtolower( $url ), PHP_URL_HOST)` to grab just the host as a string (ref. http://php.net/manual/en/function.parse-url.php).